docs/_snippets/_apt_dependencies.rst

```diff
@@ -0,0 +1,6 @@
+.. code-block:: bash
+
+	apt install \
```

Comment:
I'm not going to check dependencies, I assume these are the minimum necessary for the bot to work